copyable.gemspec

```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activerecord", "~>4.1.0"
+  spec.add_dependency "activerecord", "~>4.2"
 
   spec.add_development_dependency "database_cleaner", "~> 1.4.0"
   spec.add_development_dependency "bundler", "~> 1.6"
   spec.add_development_dependency "rake"
   spec.add_development_dependency "rspec"
-  spec.add_development_dependency "sqlite3"
+  spec.add_development_dependency "sqlite3", "~> 1.3.6"
```

Comment:
Ooh. This is a minimum? (I prefer to have as few version pins as possible, so if there's a reason for a version pin we should comment it.)

Reply:
:thinking: I think this may have been an artifact from struggling with ruby "helpfully" installing global gems that overrode everything we tried to use in bundler, which caused bundler not to run... i'll check to see if it is a minimum

Reply:
It does want a version pin, I think because of Activerecord. I commented it accordingly.